Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,52 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
-        }
-    }
-}
+            Console.WriteLine("First User");
+            Console.Write("First name: ");
+            string firstName = Console.ReadLine();
+
+            Console.Write("Last name: ");
+            string lastName = Console.ReadLine();
+
+            Console.Write("Age: ");
+            var age = Console.ReadLine();
+
+            Console.Write("Height (in cm): ");
+            var height = float.Parse(Console.ReadLine());
+
+            Console.Write("Weight (in kg): ");
+            var weight = float.Parse(Console.ReadLine());
+
+            Console.WriteLine(firstName + " " + lastName + " is " + age + " years old, their weight is " + weight + "kg and their height is " + height + " cm.");
+
+            var toMeters = height / 100;
+            var metersSquared = toMeters * toMeters;
+            var bmi = weight / metersSquared;
+            Console.WriteLine(firstName + " " + lastName + " BMI: " + bmi);
+
+            Console.WriteLine("Second User");
+            Console.Write("First name: ");
+            string firstName2 = Console.ReadLine();
+
+            Console.Write("Last name: ");
+            string lastName2 = Console.ReadLine();
+
+            Console.Write("Age: ");
+            var age2 = Console.ReadLine();
+
+            Console.Write("Height (in cm): ");
+            var height2 = float.Parse(Console.ReadLine()); ;

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this empty statement.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4842747)